astroquery/gaia/core.py

```diff
@@ -0,0 +1,561 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+=============
+Gaia TAP plus
+=============
+
+@author: Juan Carlos Segovia
+@contact: juan.carlos.segovia@sciops.esa.int
+
+European Space Astronomy Centre (ESAC)
+European Space Agency (ESA)
+
+Created on 30 jun. 2016
+
+
+"""
+
+from astroquery.tap import TapPlus
```

Comment:
I wonder whether utils  (``astroquery.utils.tap``) would be a better place for the tap module than the top level? what do you think @keflavich?